spec/factories/users.rb

```diff
@@ -3,7 +3,8 @@
 FactoryBot.define do
   factory :user do
     sequence(:username) { |n| "user#{n}" }
-    email { FFaker::Internet.safe_email(username) }
+    unconfirmed_email { FFaker::Internet.safe_email(username) }
     password { FFaker::Internet.password(10, 72) }
+    email_confirmation_token { 'token' }
```

Comment:
Split this off into an `unconfirmed_user` factory, default factory should be confirmed